src/Card/Card.tsx

```diff
@@ -21,31 +21,37 @@ export interface CardProps {
 const Card: React.FC<CardProps> = (props) => {
 
   return (
-		<Link
-			to={`/results/${props.locality}`}
-			className="card-link-wrapper"
-			style={{ textDecoration: "none" }}
-			onClick={() => {
-				props.getMatchDetails(props.lat, props.long, props.locality, props.region)
-				props.getAllDetailsData(props.lat, props.long)
-			}}
-		>
-		<article className="card-container" key={props.locality}>
-		<h2 className="card-header">{props.locality}</h2>
-			<span className="card-list">
-				{temp(props.temp)}
-				<p className="unit">&deg;F</p>
-			</span>
-			<span className='card-list'>
-				{uvIndex(props.uvi)}
-				<p className="unit">UVI</p>
-			</span>
-			{/* {aqIndex(props.aqi, props.aqiCat)}
-			<p className="unit">AQI</p> */}
-			{weatherIcon(props.icon)}
-		</article>
-		</Link>
-	)
+   <Link
+    to={`/results/${props.locality}`}
+    className="card-link-wrapper"
+    style={{ textDecoration: "none" }}
+    onClick={() => {
+     props.getMatchDetails(props.lat, props.long, props.locality, props.region);
+     props.getAllDetailsData(props.lat, props.long);
+    }}
+   >
+    <article className="card-container" key={props.locality}>
+     <h2 className="card-header">{props.locality}</h2>
+     <div className="data-numbers">
+      <span className="card-list">
+       {temp(props.temp)}
+       <p className="unit">&deg;F</p>
+      </span>
+      <span className="card-list">
+       {uvIndex(props.uvi)}
+       <p className="unit">UVI</p>
+      </span>
+      {props.aqi || props.aqiCat ? (
+       <span className="card-list">
+        {aqIndex(props.aqi, props.aqiCat)}
```

Comment:
I'm so happy you were able to get this working and displaying! Way to push through!